person.rb

```diff
@@ -20,6 +20,11 @@ def can_use_services?
     true if is_of_age? || @parent_permission
   end
 
+  def add_rental()
+    @rental.push(rental)
+    rental.person = self
+  end
```

Comment:
- [ ] When creating a `rental` in this class you should send `book` and `date` as parameters and remember The `person` itself should be sent as a parameter to create the new rental achieving the 'has-many' association you can do:
```ruby
def add_rental(book, date)
    Rental.new(date, book, self)
  end
```